File: core/src/main/java/com/maklumi/AvoidObstacle.java
package com.maklumi;

import com.badlogic.gdx.Game;

/** {@link com.badlogic.gdx.ApplicationListener} implementation shared by all platforms. */
public class AvoidObstacle extends Game {
	@Override
	public void create() {
		setScreen(new FirstScreen());
	}
}

File: core/src/main/kotlin/com/maklumi/AvoidObstacle.kt
package com.maklumi

import com.badlogic.gdx.Application
import com.badlogic.gdx.Game
import com.badlogic.gdx.Gdx
import com.maklumi.screen.FirstScreen

class AvoidObstacle : Game() {

    override fun create() {
        Gdx.app.logLevel = Application.LOG_DEBUG
        screen = FirstScreen()
    }
}